lib/atum/generation/generators/views/module.erb

```diff
@@ -72,12 +72,14 @@ module <%= @module_name %>
   def self.default_options
     {
       default_headers: {
-        <%= @default_headers.merge(
-            'User-Agent' => '#{user_agent}',
-            'Content-Type' => 'application/json'
-          ).map do |k, v|
+      <%=
+        {
+          'User-Agent' => "#{user_agent}",
+          'Content-Type' => 'application/json'
+        }.merge(@default_headers).map do |k, v|
           "\"#{k.to_s}\" => \"#{v.to_s}\""
-        end.join(",\n        ") %>
+        end.join(",\n        ")
+      %>
```

Comment:
Think this needs to be on the above line to prevent an extra line of whitespace?